README.md

```diff
@@ -47,26 +47,14 @@ The POA TokenBridge provides three operational modes:
 
 ## Initializing the monorepository
 
-Clone the repository with submodules:
+Clone the repository:
 ```bash
-git clone --recursive https://github.com/poanetwork/tokenbridge
-
-# or initialize submodules if already cloned without --recursive option:
-git submodule update --init
+git clone https://github.com/poanetwork/tokenbridge
 ```
 
-Install dependencies:
-
+Initialize submodules, install dependencies, compile the Smart Contracts:
 ```
-yarn install && yarn install:deploy
-```
-
-_**Note**: The installation should be performed with an unprivileged Linux account or with the following flag: `yarn install --unsafe-perm`. [More information](https://docs.npmjs.com/misc/scripts#user)_
```

Comment:
What will happen if `yarn initialize` will run with a privileged (e.g. `root` on Linux) account?

Reply:
I was mistaken thinking that CircleCI runs in the context of root. I will take this into account and make sure it works.

Reply:
I have added the `unsafe-perm` flag into our `initialize` script, and have confirmed that it works for both privileged and not-privileged Linux account.
So now I think the readme note is truly not-needed.

I have added a `initialize-root` CI job which makes sure that it works for root. I have set the job to run only on master, because I think we don't need to run it too often.